test/test_decision.py

```diff
@@ -91,6 +92,55 @@ def test_no_email_owner(self):
         params = decision.get_decision_parameters(FAKE_GRAPH_CONFIG, self.options)
         self.assertEqual(params["owner"], "ffxbld@noreply.mozilla.org")
 
+    @unittest.mock.patch.object(
+        GitRepository,
+        "get_commit_message",
+        unittest.mock.MagicMock(return_value="Add Foo"),
+    )
+    @unittest.mock.patch.object(
+        HgRepository,
+        "get_commit_message",
+        unittest.mock.MagicMock(return_value="Add Foo"),
+    )
+    def test_default(self):
+        """
+        Ensures `target_tasks_method` is `default` when the commit message does not contain
+        `DONTBUILD`.
+        """
+        self.options["tasks_for"] = "github-push"
+        params = decision.get_decision_parameters(FAKE_GRAPH_CONFIG, self.options)
+        self.assertEqual(params["target_tasks_method"], "default")
+
+        self.options["tasks_for"] = "hg-push"
+        params = decision.get_decision_parameters(FAKE_GRAPH_CONFIG, self.options)
+        self.assertEqual(params["target_tasks_method"], "default")
+
+    @unittest.mock.patch.object(
+        GitRepository,
+        "get_commit_message",
+        unittest.mock.MagicMock(return_value="DONTBUILD"),
+    )
+    @unittest.mock.patch.object(
+        HgRepository,
+        "get_commit_message",
+        unittest.mock.MagicMock(return_value="DONTBUILD"),
+    )
+    def test_dontbuild(self):
```

Comment:
Nit: Same rationale with this test message. This way, it'll be easier to future readers to understand these 2 tests are related. 

```suggestion
    def test_dontbuild_commit_message_yields_default_target_tasks_method(self):
```

Reply:
Thanks for taking care of them.